load/S3ZipperService.py

```diff
@@ -0,0 +1,258 @@
+import time
+import boto3
+import requests
+from datetime import datetime, timedelta, timezone
+
+# Global variable for the s3zipper base URL
+S3ZIPPER_BASE_URL = "https://api.s3zipper.com"
+MAX_WAIT_SECONDS = 3600  # 1 hour
+POLL_INTERVAL = 2
+STS_EXPIRATION_TIME = 3600  # 1 hour
```

Comment:
@bradflood I tested with the no-rate-limit credentials and the process can take up to 27 minutes. Therefore, I increased the STS expiration time.